front/src/modules/pathfinding/components/Itinerary/ItineraryV2.tsx

```diff
@@ -111,9 +111,7 @@ const ItineraryV2 = ({
               className="col my-1 text-white btn bg-info btn-sm"
               type="button"
               onClick={() =>
-                openModal(
-                  <ModalSuggestedVias suggestedOps={pathProperties.suggestedOperationalPoints} />
-                )
+                openModal(<ModalSuggestedVias suggestedOps={pathProperties.allVias} />)
```

Comment:
Can you rename the suggestedOps in ModalSuggestedVias ? 
```suggestion
                openModal(<ModalSuggestedVias suggestedVias={pathProperties.allVias} />)
```

Reply:
Done